plugins/jobs/src/js/components/form/helpers/DefaultFormData.ts

```diff
@@ -10,7 +10,8 @@ import {
 export const JobDataPlaceholders = {
   maxLaunchDelay: 3600,
   timezone: "UTC",
-  startingDeadlineSeconds: 900
+  startingDeadlineSeconds: 900,
+  concurrencyPolicy: "FORBID"
```

Comment:
Let's remove this or make the placeholder "ALLOW", as that is what the metronome API defaults to

Reply:
Okay, I made it "forbid" since the checkbox is unchecked by default.